feature-libs/my-account/organization/src/components/shared/form/currency/currency-common-form.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Currency, CurrencyService } from '@spartacus/core';
+import { Observable } from 'rxjs';
+
+@Injectable({ providedIn: 'root' })
+export class CurrencyCommonFormComponentService {
+  protected formConfiguration: { [key: string]: any } = {
```

Comment:
1. I like that customers can customize the configurations of each form part independently (i.e. simple currency form configuration can be a part of many other forms that contain currency isocode).
2. A nice-to-have benefit of component service is that the source of currencies can be also eaisly overwritten by customers

That being said, I'm not sure (2) adds much value. And (1) perhaps could be covered by a new Storefront's config. Then we would have less classes. Let's leave it for discussion.